src/helpers/routes.js

```diff
@@ -5,15 +5,16 @@ import { Provider } from 'react-redux';
 
 import App from '../components/App';
 import Error404 from '../components/errors/Error404';
-import HomePage from '../components/HomePage';
+import HomePage from '../components/pages/HomePage';
+import requestPipeline from '../components/pages/requestPipeline';
```

Comment:
Don't be working on the request pipeline inside of this story. This story was only for adding that request button. Anything you did for the request pipeline should be in it's own branch.

Reply:
@jd1388 I feel it may be ok as long as he is just making the button go to the pipeline which can just be a blank page that says pipeline.This would mean the functionality of the button is correct. However anything beyond this point is too far.